tfhe/src/core_crypto/algorithms/lwe_multi_bit_programmable_bootstrapping.rs

```diff
@@ -1680,11 +1589,25 @@ pub fn std_multi_bit_deterministic_blind_rotate_assign<Scalar, InputCont, Output
             );
 
             let dest_idx = thread_id;
-            for (lwe_mask_elements, ggsw_group) in
-                work_queue.iter().skip(thread_id).step_by(thread_count.0)
-            {
-                let (ready_for_consumer_lock, condvar, fourier_ggsw_buffer) =
-                    &fourier_multi_bit_ggsw_buffers[dest_idx];
+
+            #[allow(clippy::type_complexity)]
+            let (ready_for_consumer_lock, condvar, fourier_ggsw_buffer): &(
+                Mutex<bool>,
+                Condvar,
+                Mutex<FourierGgswCiphertext<ABox<[c64]>>>,
```

Comment:
why does it need a type annotation ?

Reply:
error[E0282]: type annotations needed for `&(_, _, _)`

Reply:
ok 🤔